packages/cypress/tests/cypress.config.ts

```diff
@@ -1,9 +1,9 @@
 import { defineConfig } from 'cypress';
-import { setupNetworkListener, onBeforeBrowserLaunch, saveArchives } from '../src/cypress-api';
+import { setupNetworkListener, onBeforeBrowserLaunch, saveArchives } from '../src';
 
 export default defineConfig({
   // needed since we use common mock images between Cypress and Playwright
-  fixturesFolder: 'fixtures',
+  fixturesFolder: '../../../dev-server/fixtures',
```

Comment:
How about `tests-server` instead of `dev-server`? That way it's clear that it's for running tests and not for doing your normal dev work. 

But I don't feel too strongly about it.

Reply:
Good point, yeah I can change it.